res/skins/Deere/style.qss

```diff
@@ -1742,6 +1742,18 @@ WPushButton[value="2"]:hover {
     border-radius: 3px;
 }
 
+WCueMenuPopup #CueRemoveButton {
+  qproperty-icon: url(:/images/library/ic_library_cross.svg);
```

Comment:
True. Quoting from an earlier conversation about this:

@Holzhaus:
>> @ronso0: Also, I somehow fear the clear button could be mistaken as Clear button for the lineedit.
>> @daschuer: The remove button is quite ambiguous. Since the dialog edits cue properties it is not clear that the x clears the whole cue.
>
>  Yes, you're right. I think this would be fixed by using the trash bin icon like this one: https://github.com/feathericons/feather/blob/master/icons/trash-2.svg
> I don't know what the policy regarding pulling in third party icons is. The icon I posted is MIT licensed, so we could just use it but need to add credits somewhere. But I guess that most skins will overwrite the icon anyway, so I'm not sure if it make sense to add this icon to the Mixxx repo in this PR.

@ronso0:
>> But I guess that most skins will overwrite the icon anyway, so I'm not sure if it make sense to add this icon to the Mixxx repo in this PR.
>
> :+1: With #2363 each skin's menus have different bg colors, so we need contrasting icons anyway that also match the color scheme.

If I understood @ronso0 correctly, it doesn't make sense to add it now, when skins have to introduce their custom versions anyway.
We can add a generic icon if we add a [default stylesheet](https://github.com/mixxxdj/mixxx/pull/2362#discussion_r353512268) though.

Reply:
I think we should not require every legacy skin to deal with it.
So let's pick a icon that is not replaced by at least one other skin.

Reply:
Ok, I implemented the default stylesheet solution in this PR and also used my dope ass inkscape skills to make a trash bin icon.